package.json

```diff
@@ -35,26 +35,26 @@
   },
   "extensionDependencies": [],
   "dependencies": {
-    "console-stamp": "^0.2.2",
-    "diff": "~3.3.0",
+    "console-stamp": "^0.2.7",
+    "diff": "^3.5.0",
     "json-rpc2": "^1.0.2",
-    "vscode-debug-logger": "^0.0.4",
-    "vscode-debugadapter": "^1.30.0",
-    "vscode-debugprotocol": "^1.11.0",
+    "vscode-debug-logger": "0.0.5",
+    "vscode-debugadapter": "^1.32.1",
+    "vscode-debugprotocol": "^1.32.0",
     "vscode-extension-telemetry": "^0.0.22",
-    "vscode-languageclient": "~4.3.0"
+    "vscode-languageclient": "^5.1.0"
   },
   "devDependencies": {
-    "@types/fs-extra": "0.0.35",
-    "@types/mocha": "^2.2.33",
+    "@types/fs-extra": "^5.0.4",
+    "@types/mocha": "^5.2.5",
     "@types/node": "^6.0.50",
-    "fs-extra": "^1.0.0",
-    "tslint": "^4.0.2",
-    "typescript": "^2.1.5",
+    "fs-extra": "^7.0.0",
+    "tslint": "^5.11.0",
+    "typescript": "^3.1.3",
     "vscode": "^1.1.21"
   },
   "engines": {
-    "vscode": "^1.22.0"
+    "vscode": "^1.26.0"
```

Comment:
Any particular reason for this change?
This change will result in the next update to the Go extension not being available to anyone who is on versions of VS Code older than 1.26.0

Reply:
There where typing issues with one of the dependencies with the 1.22 version, I based my solution on this https://github.com/Microsoft/language-server-protocol/issues/556#issuecomment-416884011 comment

Reply:
In that case, lets not update any of the `vscode-` dependencies just yet.

I usually prefer to wait till we are atleast 3 versions away from the minimum specified in the `engine` property. Unless, of course there is a bug fix that we need to take in immediately.